adding EWK weights to weightF and QCD weight to W+jets and Znunu+Jets…

benjaminkreis opened the pull request:
adding EWK weights to weightF and QCD weight to W+jets and Znunu+Jets HT bins

benjaminkreis committed:
adding EWK weights to weightF and QCD weight to W+jets and Znunu+Jets HT bins

jstupak commented:
I think the cross sections for W+jets is now too large by a factor 1.23, since the k-factor is included in the cross section (a double counting of the k-factor).

While we are at it, can we switch to the HT binned Z->ll samples for consistency?

benjaminkreis committed:
remove W+Jets 1.23 k factor, add HT-binned DY as samples but don't use yet

benjaminkreis commented:
I removed the 1.23 k factor and added the HT-binned DY samples in samples_no split

HOWEVER I did not finish the setup of the HT binned DY.  Inclusive is still being used.  Switching requires some tricky changes to "general" that will take some time for me to figure out.

benjaminkreis committed:
Properly add V15 and V15a MET data sample by excluding V15 Prompt Reco.  Still needs to be done for Zmumu and Wmunu areas.
